Musicplayer.css

```diff
@@ -67,62 +41,152 @@ li{
 }
 
 
+
 nav a{
 	display:block;
 	width:100px;
 	padding: 20px 15px;
 	text-decoration:none;
-	font-family:Arial;
-	color:black;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:bold;
+	color:white;
 	text-align: center;
 }
 
 nav a:hover{
-	background:purple;
+	background:black;
```

Comment:
21/9: I change the hover background color to black because the hover background color was just be the place holder to make sure it work. So when I decided the color theme I start to apply color black to be the hover background color because it was one of the color themes that I decided to apply and also it can display a great contrast with navy blue and white.